### PR TITLE
Add Tendermint liquidity

### PR DESCRIPTION
**Pull request by patiee**

### CODE REVIEW

**Review by lavabyrd — CHANGES_REQUESTED** [review]
This looks ok. We'd need to fix the conflicts first though but seems alright to me

File: liquidity.v1.proto [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+        example : "\"denomB\"",
+      }];
+  cosmos.v1.Coin offer_coin_fee = 6 [(gogoproto.nullable) = false, (gogoproto.moretags) = "yaml:\"offer_coin_fee\"",
+                                     (grpc.gateway.protoc_gen_openapiv2.options.openapiv2_field) =
+                                         {example : "{\"denom\": \"denomX\", \"amount\": \"5000\"}", format : "sdk.Coin"}];
+  string         order_price =
+      7 [(gogoproto.moretags) = "yaml:\"order_price\"", (gogoproto.customtype) = "github.com/cosmos/cosmos-sdk/types.Dec",
+         (gogoproto.nullable)                                        = false,
+         (grpc.gateway.protoc_gen_openapiv2.options.openapiv2_field) = {example : "\"1.1\"", format : "sdk.Dec"}];
+}
```
Comment (lavabyrd) [inline liquidity.v1.proto:37]: the formatting seems super strange here

**Review by lavabyrd — APPROVED** [review]

**Review by lavabyrd — APPROVED** [review]
just some strange formatting

btw not sure if your protobuf formatting will work here as it seems to reference codec.proto, we can clean it up later but lets get a task open so we don't ignore it

File: liquidity.v1.proto (hunk 1 above)
Comment (patiee) [reply liquidity.v1.proto:37]: haha that's right :D hmm maybe I will delete new lines and try to format it again

Comment (patiee) [reply liquidity.v1.proto:37]: it doesn't changed anything :C we can think about improving formatting

**Pull request merged**